client/ts/src/market.ts

```diff
@@ -180,6 +180,32 @@ export class Market {
     return toNum(isBase ? seat.baseBalance : seat.quoteBalance);
   }
 
+  /**
+   * Get the amount in tokens of balance that is deposited on the exchange, does
+   * not include tokens currently in open orders.
```

Comment:
Also remove the atoms one. Make so users never can have the chance to mess up by using it

Reply:
remove the entire method or just fix the comment?

Reply:
fixed the comment

Reply:
remove the entire function. can do later

Reply:
fixed in the follow up pr